gif_sizer/__main__.py

```diff
@@ -0,0 +1,4 @@
+from .parser import parse_command
+
```

Comment:
Main file does not need to have name '\_\_main\_\_'
\_\_name\_\_ parameter will have value "\_\_main\_\_" if it's the 'root' file executed by python.
So if file was imported from another python script - \_\_name\_\_ != "\_\_main\_\_"

Sorry.. you probably already know that :)

Reply:
The idea here is to have a python package eventually :)

Look here how the package can be structured using `__main__.py` : [https://docs.python.org/3/library/__main__.html#main-py-in-python-packages](https://docs.python.org/3/library/__main__.html#main-py-in-python-packages)

It has nothing to do with `if __name__ == "__main__":` in this case ;)

Reply:
Nice, I’ve never used it like that :)

Reply:
Sorry but this simple test proves the opposite, the package without `if __name__ == "__main__"` statement won't execute the main() function 😢 :

<img width="450" alt="image" src="https://user-images.githubusercontent.com/35875048/214975638-125e9c5f-dec5-4f0c-9b4e-23788d449118.png">

Maybe I didn't catch what you meant? 😄

Reply:
You still nead to call the method `main`:
```
def main():
    print("Just a test")
    
main() 
```

But condition `if __name__ == "__main__"` is not needed in this case.

Reply:
Yes, python allows such call but it is against the convention. This is intended to be a package and act as an executable program, therefore an entrypoint is needed and name-main statement is needed to do so. This is very standard design pattern in python :)